virttest/utils_test/qemu.py

```diff
@@ -1842,3 +1842,308 @@ def get_all_memorys(self, vm):
         session = self.get_session(vm)
         output = session.cmd_output_safe(cmd, timeout=90)
         return set([_ for _ in output.splitlines() if _])
+
+
+class UsbDevTest(object):
+
+    """
+    Provide basic test for usb devices
+    """
+
+    def __init__(self, test, params, env):
+        self.params = params
+        self.test = test
+        self.env = env
+
+    def check_usb_dev(self, usb_type, info_usb_name,
+                      vp_id, vendor, product, vm):
```

Comment:
if vm is used for many times, maybe it's better to initialized it in __init__().